bookwyrm/tests/activitypub/test_author.py

```diff
@@ -1,15 +1,17 @@
 import datetime
 
+from unittest.mock import patch
```

Comment:
The rest of this PR is tests, tests, tests. It allowed me to refine the trigger for the tasks.

Seeing how much code is changed only to patch the function made me realize how much computing power the feature could hog.

Also, it might be simpler if there's only one task, see one of my above  comments on the subject.